tsdb/compact_test.go

```diff
@@ -163,7 +163,7 @@ func TestNoPanicFor0Tombstones(t *testing.T) {
 }
 
 func TestLeveledCompactor_plan(t *testing.T) {
-	// This mimicks our default ExponentialBlockRanges with min block size equals to 20.
+	// This mimics our default ExponentialBlockRanges with min block size equals to 20.
```

Comment:
Note to other reviewers: We had this already, and it was concluded that “mimicks” is fine. However, it seems to be a fairly uncommon spelling (Wiktionary lists it as “alternative” form, while the Merriam Webster doesn't list it at all). To avoid more of the spelling-“correction” PRs about this, I'd be inclined to accept this change and be done with it.